docs/java/getting-started.md

```diff
@@ -100,6 +107,81 @@ application  cx-server  integration-tests  Jenkinsfile  manifest.yml  pom.xml  u
 
 **Congratulations! You've just configured your application with Cloud SDK for Java.**
 
+## Integrate the Cloud SDK for Java into your Project
+
+To get started include the _SDK BOM_ in the _dependency management_ of your project:
+
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.sap.cloud.sdk</groupId>
+            <artifactId>sdk-bom</artifactId>
+            <version>use-latest-version-here</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>        
+</dependencyManagement>
+```
+
+If your application is running on SAP Cloud Platform please also include either:
+
+```xml
+<dependency>
+    <groupId>com.sap.cloud.sdk.cloudplatform</groupId>
+    <artifactId>scp-cf</artifactId>
+</dependency>
+```
+
+or:
+
+```xml
+<dependency>
+    <groupId>com.sap.cloud.sdk.cloudplatform</groupId>
+    <artifactId>scp-neo</artifactId>
+</dependency>
+```
+
+If you want to connect to an S/4HANA system via the OData protocol you should also add a dependency to the client library of the SDK:
+
+```xml
+<dependency>
+    <groupId>com.sap.cloud.sdk.s4hana</groupId>
+    <artifactId>s4hana-all</artifactId>
+</dependency>
+```
+
+### Framework integration
+
+In general, the Cloud SDK for Java integrates natively into the [Spring Boot](https://spring.io/projects/spring-boot) and [TomEE](https://tomee.apache.org/) frameworks.
+
+In particular the [SDK provides listeners](features/multi-tenancy/thread-context.md) that can extract tenant and principal information from an incoming request. To ensure these listeners are present please configure your project accordingly.
+
+<Tabs groupId="frameworks" defaultValue="spring" values={[
+{ label: 'Spring', value: 'spring', },
+{ label: 'TomEE', value: 'tomee', }]}>
+
+<TabItem value="spring">
+
+For a Spring based project please ensure that the application is annotated to scan for components of the SDK:
+
+```java
+@ComponentScan({"com.sap.cloud.sdk", "your.own.package"})
+@ServletComponentScan({"com.sap.cloud.sdk", "your.own.package"})
+```
+
+Check the logs on application startup to ensure the listeners got registered. Also please check the spring version declared in the SDK BOM doesn't clash with your version of Spring.
+
+</TabItem>
+<TabItem value="tomee">
+
+For a TomEE based project the filters should be registered automatically. They are part of the `servlet` module which comes into the dependency tree through `scp-cf` or `scp-neo` automatically. Check the logs on application startup to ensure the listeners are being registered.
+
+</TabItem>
+</Tabs>
+
```

Comment:
Idea, optional: add an additional section on adding the usage analytics plugin of the SDK to the pom (basically what is part of our archetypes out of the box). Also helps with support via the output command of the plugin, which prints helpful infos about the project structure etc.

Also see
* https://github.com/SAP/cloud-s4-sdk-pipeline/blob/master/doc/operations/analytics.md
* https://blogs.sap.com/2018/10/23/usage-analytics-s4sdk/

@artemkovalyov something for the documentation backlog?

Reply:
Good idea, done ✅

Reply:
@sacnl, I'll need a bit more input on how it helps with support, but encouraging sending analytics back ESPECIALLY for internal projects is a must. Thanks for these suggestions! We can link a blog by Sander or make a part of it a quick hint on enabling analytics if I understood you correctly.